.github/workflows/ci-pull-request.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
     needs: lint
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/pegasus-kv/thirdparties-bin:ubuntu1804
+      image: apachepegasus/thirdparties-bin:ubuntu1804
```

Comment:
Actually It's recommended to upload the image to ghcr to benefit from the internal network between github action and ghcr.  Or else you would have to pay dockerhub for the download when it exceeds a certain threshold.

Reply:
I don't want maintain multi docker images hub, I actually test it network, if it is not too slow, I think it's no problem.

Reply:
Oh, I see, dockerhub limit `pull time`, I will close it, and change pegasus-docker to `ghcr` in later